src/code_800AF9B0.c

```diff
@@ -2987,7 +2995,7 @@ void validate_save_data(void) {
             gSaveDataSoundMode = gSaveDataSoundModeBackup;
             D_8018ED16 = compute_save_data_checksum_backup_1();
             D_8018ED17 = compute_save_data_checksum_backup_2();
-            osEepromLongWrite(&gSIEventMesgQueue, (((u32) &D_8018ED10 - (u32) &D_8018EB90) >> 3), &D_8018ED10, 8);
+            osEepromLongWrite(&gSIEventMesgQueue, (((u32) &D_8018ED10 - (u32) &D_8018EB90) >> 3), (u8*) &D_8018ED10, 8);
```

Comment:
So I added the union, but this got me thinking... 
This call is meant to write 8 bytes starting at D_8018ED10 (4 bytes for the grandPrixPoints, 1 for the soundMode, and 3 for the checkSum), but with the union D_8018ED10 is now 4 bytes on its own?

Or does these lines in asm/bss_8001C4D0.s mean that D_8018ED11 will still be one byte after D_8018ED10?
glabel D_8018ED10
.skip 1

Reply:
Quick investigation looking at build/us/mk64.us.map and it appears making D_8018ED10 a union of size 4 is fine